client/lobby/SelectionTab.cpp

```diff
@@ -353,7 +359,7 @@ void SelectionTab::showPopupWindow(const Point & cursorPosition)
 		if(curItems[py]->date != "")
 			text += boost::str(boost::format("\r\n\r\n%1%:\r\n%2%") % CGI->generaltexth->translate("vcmi.lobby.creationDate") % curItems[py]->date);
 
-		CRClickPopup::createAndPush(text);
+		GH.windows().createAndPushWindow<CMapInfoTooltipBox>(text, ResourceID(curItems[py]->fileURI), tabType);
```

Comment:
I suggest to add feature flag in game config in case to disable if somebody will have performance issues (for example, maps could be stored somewhere on network path)

Reply:
Not against it, but don't see much point either - we still need to load header for all maps to populate list and even XL maps are quite small, around 100kb